.github/workflows/build.yml

```diff
@@ -65,13 +64,8 @@ jobs:
         if: steps.clojure-deps.outputs.cache-hit != 'true'
         run: clojure -A:cljs -P
 
-      - name: Set up clj-kondo
```

Comment:
@andelf I removed this as I'm able to run clj-kondo profile on the older version. I've filed https://github.com/clj-kondo/clj-kondo/issues/1536 for the hang I'm seeing on latest clj-kondo

Reply:
Hang issue resolved. sci dependency was biting us. Created https://quire.io/w/Logseq-_team_dedicated_to_sport_of_knowledge/107/ for upgrading dependencies more regularly